index.html

```diff
@@ -1199,82 +1272,80 @@ <h4>
           representation of the {{GeolocationCoordinates}} object.</ins>
         </p>
       </section>
-      <section>
+      <section data-cite="infra">
         <h2>
           Constructing a `GeolocationPosition`
         </h2>
+        <aside class="correction" id="c4">
+          <span class="marker">Candidate Correction:</span> Constructor now
+          takes a [=map=] of position data, a timestamp, and a boolean
+          indicating high accuracy as arguments. We itereate over the map to
+          set the attributes of the {{GeolocationCoordinates}}.
+        </aside>
         <p>
-          <dfn>A new `GeolocationPosition`</dfn> is constructed with
+          <dfn>A new `GeolocationPosition`</dfn> is constructed with [=map=]
+          |positionData|, {{EpochTimeStamp}} |timestamp:EpochTimeStamp| and
+          boolean |isHighAccuracy| by performing the following steps:
+        </p><ins cite="#c4">
+        <ol class="algorithm">
+          <li>Let |coords:GeolocationCoordinates| be a newly created
+          {{GeolocationCoordinates}} instance.
+          </li>
+          <li>[=map/For Each=] |key| → |value| in |positionData|:
+            <ol>
+              <li>Set |coords|'s attribute named |key| to |value|.
+              </li>
+            </ol>
+          </li>
+          <li>Return a newly created {{GeolocationPosition}} instance with its
+          {{GeolocationPosition/coords}} attribute initialized to |coords| and
+          {{GeolocationPosition/timestamp}} attribute initialized to
+          |timestamp|, and its {{GeolocationPosition/[[isHighAccuracy]]}}
+          internal slot set to |isHighAccuracy|.
+          </li>
+        </ol></ins> <del cite="#c4">
+        <p>
+          <strong>A new `GeolocationPosition`</strong> is constructed with
           {{EpochTimeStamp}} |timestamp:EpochTimeStamp| and boolean
           |isHighAccuracy| by performing the following steps:
         </p>
```

Comment:
Again here, I'd rather have finer-grained `<ins>` and `<del>` sections to show readers what has changed. I think you can `<ins>` the new constructor parameter without fully replacing its paragraph; fully replace the series of "Initialize X to Y" steps with your loop; and keep the "Return a newly created" step as-is.

As a really nit-picky note, WebIDL actually says that [attributes](https://webidl.spec.whatwg.org/#idl-attributes) aren't things that specs can initialize or set the value of: they have [getters](https://webidl.spec.whatwg.org/#getter-steps), which return values that are often held in a struct. But I think your description here is clear enough that nobody will implement it non-interoperably, so it doesn't need to change.

Reply:
Discussed with Elika and David Singer and we prefer the larger ins/del as otherwise the ins/dels become unmanageable. 

> As a really nit-picky note, WebIDL actually says that [attributes](https://webidl.spec.whatwg.org/#idl-attributes) aren't things that specs can initialize or set the value of 

yeah, I know ... we should use internal slots here (or hold the map internally in that slot). However, I'll update that in the future as it will add a lot of changes.

Reply:
Works for me. The arrangement of corrections is a Process issue for which Elika is particularly authoritative.

Reply:
If we eventually get tooling to do this, we will probably to the fine-grained changes.